### PR TITLE
DB-9855 Use no-op constraint checker in the write pipeline for sys tables 

### PR DESCRIPTION
**Pull request by arnaud-splice**

### CODE REVIEW

File: pipeline_api/src/main/java/com/splicemachine/pipeline/constraint/NoOpConstraint.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import com.splicemachine.si.api.server.ConstraintChecker;
+import com.splicemachine.si.api.txn.TxnView;
+import com.splicemachine.utils.ByteSlice;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * A Unique Constraint.
```
Comment (dgomezferro) [inline pipeline_api/src/main/java/com/splicemachine/pipeline/constraint/NoOpConstraint.java:30]: Update / remove comment

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1256/

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Pull request merged**